### ISSUE TITLE
Peer Review - Linux

### ISSUE THREAD

**Issue by griffinhadfield** [OpenLiberty/draft-guide-operator-openshift#6]
Functionality
---
- [x] Check that the guide has working instructions and sample code
- [ ] If any URL visits, try them on different browsers: Firefox, Chrome, Safari
- [x] If any URL visits, try `curl` command where applicable 

Formatting & Presentation
---

`README.adoc` checks:
- [x] Ensure that command substitutions use the `$(cmd)` format and not ``` `cmd` ```
- [ ] Ensure that the console output of the integrated tests do not include the `[INFO]` tab
- [x] Check the quality and presentation of guide according to [Structure and Style Guideline](https://github.com/OpenLiberty/draft-guides-template/wiki/Guidelines-for-Structure-and-Styles)
- [x] Check that the pom.xml, server.xml, etc files are clean
    - [x] Remove any dependencies and features that are not required
    - [x] Ensure that the parent pom.xml files do not contain any dependencies, and all dependencies are specified in the children pom.xml files
- [x] Examples with the right outcomes are provided
- [x] Example with the wrong outcomes, if any, are provided

`pom.xml` checks (if files are present):
- [x] Check that the file follows the template found here: https://github.com/OpenLiberty/draft-guides-template/blob/master/finish/pom.xml
- [x] Check that the following dependencies are included in all of the `pom.xml` files:
```
<!-- Provided dependencies -->
<dependency>
  <groupId>jakarta.platform</groupId>
  <artifactId>jakarta.jakartaee-api</artifactId>
  <version>8.0.0</version>
  <scope>provided</scope>
</dependency>

<dependency>
  <groupId>org.eclipse.microprofile</groupId
  <artifactId>microprofile</artifactId>
  <version>3.3</version>
  <type>pom</type>
  <scope>provided</scope>
</dependency>
```
- [x]  Check that the versions of plugins are directly provided and not fed in by a file property.
  - Not formatted like so:
```
    <properties>
        <!-- Plugins -->
        <version.liberty-maven-plugin>3.2</version.liberty-maven-plugin>
    </properties>

    <!-- Liberty plugin -->
    <plugin>
        <groupId>io.openliberty.tools</groupId>
        <artifactId>liberty-maven-plugin</artifactId>
        <version>${version.liberty-maven-plugin}</version>
    </plugin>
```
  - Should be formatted like so:
```
    <!-- Liberty plugin -->
    <plugin>
        <groupId>io.openliberty.tools</groupId>
        <artifactId>liberty-maven-plugin</artifactId>
        <version>3.2</version>
    </plugin>
```
- [x] Check that there are 4 spaces per indent for proper formatting
- [x] Ensure that the guide is using the latest version of the `liberty-maven-plugin` or `liberty-gradle-plugin` where applicable

Overall checks:
- [x] Check the consistency of guide with the template and other guides
- [x] Check the quality of code according to the best coding practices
- [x] Check that all licensing statements are properly stated in all files, with the correct year (Should be present in all Java files + the `index.html`)
- [x] Check that the directories are properly structured
- [x] Check that some of these `page-tags` are used in a guide: `MicroProfile, Maven, Docker, Kubernetes, Gradle, Java EE, Security, Cloud`. Only these tags are visible on the website. Latest list [here](https://github.com/OpenLiberty/openliberty.io/blob/master/src/main/content/_includes/visible-tags.liquid). 
- [x] Check the `attribution` statement is accurate for the guide
- [x] Verify the `travisTest.sh` script, if any, is accurate and consistent with other guides

---

Guide’s contributor’s (if available, otherwise peer tester’s) responsibility: 
- [ ] Additional tests where applicable: 
    - [ ] Define test coverage and review with team (including guide contributor, if available)
    - [ ] Define detail test cases 
    - [ ] Consider corner cases targeting the specific guide
    - [ ] Consider corner cases UI tests
    - [ ] Consider testing URL on all browsers, ie, FF, Chrome, Safari
    - [ ] Consider testing the `curl` command for URL visits
- [ ] Consider building with both Maven and Gradle build tools 
- [x] Testing with different IDEs, ie, Atom, Eclipse (Optional: VS.code, IntelliJ, Microclimate)
- [ ] Run Acrolinx Checker on draft (above 70 score approximately)
- [ ] Consider SEO title and description for the guides 
- [ ] Ensure automated test is enable, set up with Travis CI, and able to schedule tests 
- [x] Run `diff -r start/ finish/` and there's no differences
- [ ] Ensure that the automation tests are able to run when PR is created

Peer Tester’s responsibility: 
- [x] Check the appearance of the guide on test site for the following items: 
    - [x] Table of contents 
    - [x] Headings
    - [x] Paragraphs
    - [x] code snippets
    - [x] outputs
    - [x] links
    - [x] hotspots 
- [x] Test the guide end-to-end with working instruction and sample code
- [ ] Perform all the defined test cases

**Comment by griffinhadfield** [comment]
<img width="805" alt="Screenshot 2020-05-15 at 9 38 53 AM" src="https://user-images.githubusercontent.com/38739044/82056983-b38c6100-9690-11ea-8d8a-86cab63882a4.png">
- The output from the guide doesn't seem to closely match the output from the console. 
<img width="1276" alt="Screenshot 2020-05-15 at 9 41 18 AM" src="https://user-images.githubusercontent.com/38739044/82056991-b5562480-9690-11ea-91a2-d2bde1d36a9f.png">

No "Push Successful" 
<img width="1276" alt="Screenshot 2020-05-15 at 9 48 33 AM" src="https://user-images.githubusercontent.com/38739044/82057411-4b8a4a80-9691-11ea-995a-f0a133f0c3ed.png">
<img width="882" alt="Screenshot 2020-05-15 at 9 48 38 AM" src="https://user-images.githubusercontent.com/38739044/82057416-4deca480-9691-11ea-8c9a-0c030aa3552b.png">

- `oc get routes` gives `No resources found in guide namespace.`

<img width="774" alt="image" src="https://user-images.githubusercontent.com/38739044/82060836-e553f680-9695-11ea-9b24-4bdc5c1daede.png">

- For the output given above, mine says configured not created

- In the first sentence of the last paragraph under "What You'll Learn" there should be a comma after inventory. 

- First sentence of "Installing the Open Liberty Operator" should be "Login to your OpenShift Cluster" not "Log into your OpenShift Cluster"

**Comment by griffinhadfield** [comment]
- [ ] Perhaps the `deploy.yaml` file should be using 4 spaces per indent to match the other files

**Comment by griffinhadfield** [comment]
- [ ]`start/models/pom.xml` features old `jakarta.jakartaee-web-api` artifactID and is missing MicroProfile dependency

**Comment by griffinhadfield** [comment]
- [ ] Hotspot for "Accessing the inventory application" switch back to build.yaml. I'm not sure if that was meant to happen.

**Comment by MaiHameed** [comment]
> My outputs (second image below) do not say "Push Successful" as indicated in the guide (first image below)

The build takes a while to finish, so the message would only show up once you check the logs after it's finished. I made it a little more clear in the README with the following sentence:

> Wait until the build completes. You will know it is complete once you see the `Push successful` message. Run the following command to view the image streams:

> oc get routes gives No resources found in guide namespace.

This might be because the builds haven't fully completed before you check for routes, and they haven't been exposed yet.

> Perhaps the build.yaml and deploy.yaml files should be using 4 spaces per indent to match the other files

Suggested syntax recommends 2 spaces in YAML files

> Hotspot for "Accessing the inventory application" switches back to build.yaml. I'm not sure if that was meant to happen.

Seems to be a UI issue, not sure I can fix this

**Comment by MaiHameed** [comment]
Addressed with #10
